packages/modal-ui/src/views/wcm-connect-wallet-view/index.ts

```diff
@@ -9,9 +9,12 @@ export class WcmConnectWalletView extends LitElement {
 
   // -- private ------------------------------------------------------ //
   private viewTemplate() {
-    if (CoreUtil.isAndroid()) {
-      return html`<wcm-android-wallet-selection></wcm-android-wallet-selection>`
-    }
+    // Disabled custom android deeplink handling because it doesn't work in telegram mini apps
+    /*
+     * If (CoreUtil.isAndroid()) {
+     *   return html`<wcm-android-wallet-selection></wcm-android-wallet-selection>`
+     * }
+     */
```

Comment:
What's up with this commented out code ? Shouldn't we keep it for non TG apps ?

Reply:
+